samples/Linux/2023.FreeDownloadManager/freedownloadmanager.sdiff

```diff
@@ -1,4 +1,4 @@
-*** changed: Linux/2023.FreeDownloadManager/freedownloadmanager_infected_postinst
+>>> moved: Linux/2023.FreeDownloadManager/freedownloadmanager_clear_postinst -> Linux/2023.FreeDownloadManager/freedownloadmanager_infected_postinst (score: 0.938235)
```

Comment:
When diffing two binaries, it shouldn't be considered a move; just a diff between two programs. 'Moved' should only be considered if directory trees are diffed.

Reply:
Makes sense.

I'll land the original changes as-is and then spend some more time overhauling the diff behavior. I don't want this scope creep to hold up the next release.

Reply:
Moved back to `filepath.Rel` in [`55a9ed2` (#356)](https://github.com/chainguard-dev/bincapz/pull/356/commits/55a9ed242264ffefb9ab919d59fac16d4bde14ae).